google/resource_bigquery_table_test.go

```diff
@@ -236,6 +267,96 @@ EOH
 }`, datasetID, tableID)
 }
 
+func testAccBigQueryTableKms(projectId, projectOrg, projectBillingAccount, keyRingName, cryptoKeyName, datasetID, tableID string) string {
+	return fmt.Sprintf(`
+resource "google_project" "acceptance" {
+	name            = "%s"
+	project_id      = "%s"
+	org_id          = "%s"
+	billing_account = "%s"
+}
+
+resource "google_project_services" "acceptance" {
+	project = "${google_project.acceptance.project_id}"
+
+	services = [
+	  "cloudkms.googleapis.com",
+		"bigquery-json.googleapis.com",
```

Comment:
nit: tabs

Reply:
I'm gonna disagree and say nit: spaces.  :stuck_out_tongue: 

Fixed it, though!